src/main/java/frc/robot/subsystems/AlgaeGripper.java

```diff
@@ -30,6 +34,10 @@ public boolean hasAlgae(){
           return !digitalInput.get();
      }
 
+     public double getVelocity(){
```

Comment:
```suggestion
     public double getVelocityRpm(){
```

always specify measurement units